### PR TITLE
Bottom bar + UI improvements

### PR DESCRIPTION
**Pull request by Piteryo**
Added Splash screen, fixed colors of status bar etc.

### CODE REVIEW

**Comment by bruce-willis** [comment]
@Piteryo, it's very difficult to see the `real changes`, when they're together with just `refactoring`. It will be much more better if u split them in 2 different pull requests next time

File: DotNetRu.Android/Renderers/BottomTabbedPageRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[assembly: ExportRenderer(typeof(BottomTabbedPage), typeof(BottomTabbedPageRenderer))]
+
+namespace DotNetRu.Droid.Renderers
+{
+    public class BottomTabbedPageRenderer : BottomTabbedRenderer
+    {
+        public BottomTabbedPageRenderer()
+        {
+            MessagingCenter.Subscribe<SettingsPage>(this, "UIUpdate", model => { this.HandlePagesChanged(); });
```
Comment (bruce-willis) [inline DotNetRu.Android/Renderers/BottomTabbedPageRenderer.cs]: Please move `UIUpdate` to message keys

File: DotNetRu.Android/Resources/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<resources>
+  <string name="news">Новости</string>
```
Comment (bruce-willis) [inline DotNetRu.Android/Resources/values-ru/strings.xml]: Do we need this? It's seemed that we delete it in last pr

File: DotNetRu.Clients.Portable/ViewModel/ViewModelBase.cs [hunk 3]
```diff
@@ -43,7 +43,15 @@ public ViewModelBase(INavigation navigation = null)
             this.Navigation = navigation;
             Resources = new LocalizedResources(typeof(ApplicationResources.AppResources), CurrentLanguage);
         }
+        //public new void OnPropertyChanged([CallerMemberName]string property = null)
```
Comment (bruce-willis) [inline DotNetRu.Clients.Portable/ViewModel/ViewModelBase.cs]: Again, we've already delete it in last pr

File: DotNetRu.Android/MainActivity.cs [hunk 4]
```diff
@@ -84,15 +84,18 @@ public MainActivity()
         {
             LocaleUtils.UpdateConfig(this);
         }
+
         public static MainActivity Current { get; }
 
         // GoogleApiClient client;
         protected override void OnCreate(Bundle savedInstanceState)
         {
+            SetTheme(Resource.Style.MyTheme);
```
Comment (pfedotovsky) [inline DotNetRu.Android/MainActivity.cs]: Why it's needed? This should be applied by default

File: DotNetRu.Android/Resources/menu/nav_menu.xml [hunk 5]
```diff
@@ -1,33 +1,34 @@
-﻿<?xml version="1.0" encoding="UTF-8" ?>
+﻿<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pfedotovsky) [inline DotNetRu.Android/Resources/menu/nav_menu.xml]: Is this file needed?

File: DotNetRu.Clients.UI/App.xaml.cs [hunk 6]
```diff
@@ -37,21 +38,22 @@ public App()
...
-                case Device.iOS:
-                    this.MainPage = new EvolveNavigationPage(new RootPageiOS());
-                    break;
-                case Device.UWP:
-                    this.MainPage = new RootPageWindows();
-                    break;
-                default:
-                    throw new NotImplementedException();
-            }
+            //switch (Device.RuntimePlatform)
```
Comment (pfedotovsky) [inline DotNetRu.Clients.UI/App.xaml.cs]: Please remove - it's no longer needed. BTW, is BottomTabbedPage supported on Windows 10?

File: DotNetRu.Clients.UI/Pages/Home/NewsPage.xaml [hunk 7]
```diff
@@ -9,7 +9,7 @@
     x:Class="XamarinEvolve.Clients.UI.NewsPage"
     x:Name="NewsScreen"
     Title="{Binding Resources[News]}"
-    Icon="tab_feed.png"
+    Icon="menu_feed.png"
```
Comment (pfedotovsky) [inline DotNetRu.Clients.UI/Pages/Home/NewsPage.xaml]: Why the icon was changed?

File: DotNetRu.Android/MainActivity.cs (hunk 4 above)
Comment (bruce-willis) [reply DotNetRu.Android/MainActivity.cs]: Exactly not. App is launched with `Theme = "@style/SplashTheme"` (see [line 34](https://github.com/DotNetRu/App/pull/55/files/82374bd5afe9ecf6c0763fced388baadc2d0de3b#diff-e8d186313ce687a878a76623be2f8b59R34)) and then set theme to main. See [this arcticle](https://android.jlelse.eu/launch-screen-in-android-the-right-way-aca7e8c31f52) for more information

Comment (pfedotovsky) [reply DotNetRu.Android/MainActivity.cs]: OK, I see, thanks!

File: DotNetRu.Android/Resources/menu/nav_menu.xml (hunk 5 above)
Comment (bruce-willis) [reply DotNetRu.Android/Resources/menu/nav_menu.xml]: No, it's part of bottom menu. I think we should delete it and all related things in separate pr (with other not needed things, like pages for uwp)

Comment (pfedotovsky) [reply DotNetRu.Android/Resources/menu/nav_menu.xml]: Ok

File: DotNetRu.Clients.UI/App.xaml.cs (hunk 6 above)
Comment (Piteryo) [reply DotNetRu.Clients.UI/App.xaml.cs]: it will render in default tabbed page on windows

File: DotNetRu.iOS/DotNetRu.iOS.csproj [hunk 8]
```diff
@@ -428,45 +428,59 @@
   </ItemGroup>
   <ItemGroup>
     <ImageAsset Include="Resources\Media.xcassets\AppIcons.appiconset\Contents.json">
+      <Visible>false</Visible>
```
Comment (pfedotovsky) [inline DotNetRu.iOS/DotNetRu.iOS.csproj]: Can you explain why it's needed?

File: DotNetRu.Android/DotNetRu.Android.csproj [hunk 9]
```diff
@@ -121,6 +121,15 @@
     <Reference Include="MvvmHelpers, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\Refractored.MvvmHelpers.1.3.0\lib\netstandard1.0\MvvmHelpers.dll</HintPath>
     </Reference>
+    <Reference Include="Naxam.BottomTabbedPage.Forms, Version=0.1.0.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (pfedotovsky) [inline DotNetRu.Android/DotNetRu.Android.csproj]: No longer needed

Comment (Piteryo) [reply DotNetRu.Android/DotNetRu.Android.csproj]: Why? It is reference to nuget package
